src/main/java/io/jenkins/plugins/sprp/PipelineSnippetGenerator.java

```diff
@@ -121,7 +121,7 @@ public String getPublishReportSnippet(ArrayList<String> paths){
         StringBuilder snippet = new StringBuilder();
 
         for(String p: paths)
-            snippet.append("junit '").append(p).append("'\n");
+            snippet.append("junit testResults: '").append(p).append(", allowEmptyResults: true").append("'\n");
```

Comment:
these changes here seem unrelated to getting a branch name and therefor should be in a different PR 
(also note the commented code that shouldn't be merged...)